lib/API/Oauth.dart

```diff
@@ -233,13 +207,13 @@ abstract class Auth {
 
     // Check if the token is not expired
     // if (_token != null) {
-    if ((_token != null) || (_token == "null")) {
+    if (_token != null && _token != "null") {
```

Comment:
I think this should be a logical AND operator, and we'd need to flip the second clause? We supposed to test here if the token is not null?

Reply:
Also removing some unnecessaary parantheses

Reply:
You are right, it should be a logical AND.
But I am not sure that "null" is a value that is returned.

Reply:
Yes, neither do I know if "null" can actually be a valid value, but the code can prepare for it. The change below is another spot where "null" is involved. I still haven't tested this fully, I hope the merge won't screw up anything.